### PR TITLE
Implements a progress bar on external video for viewers.

### PR DESCRIPTION
**Pull request by gabriellpr**
The viewer can't see if the shared video is ending or in the middle.
IMG-20220317-WA0009(1)

This PR adds a progress bar for the viewers, so they can follow the progress of the video.
IMG-20220317-WA0008(1)

### CODE REVIEW

**Comment by antobinary** [comment]
Hi @gabriellpr ! Thank you for your first contribution!
@TiagoJacobs we don't need a CLA, right?

**Review by ramonlsouza — APPROVED** [review]

**Comment by prlanzarin** [comment]
Have you considered making the progress bar visible only on hover (desktop)/touch (mobile)?
That would make it consistent with the current volume control and some common players on web browsers (eg youtube, vimeo).

File: bigbluebutton-html5/imports/api/meetings/server/modifiers/addMeeting.js [hunk 1]
```diff
@@ -120,7 +120,7 @@ export default function addMeeting(meeting) {
     systemProps: {
       html5InstanceId: Number,
     },
-  });
+  });
```
Comment (prlanzarin) [inline bigbluebutton-html5/imports/api/meetings/server/modifiers/addMeeting.js:123]: Unrelated change (alongside addPresentation.js#L52)

**Comment by gabriellpr** [comment]
Thanks for the suggestion, I will implement it and remove the white space.

**Review by prlanzarin — APPROVED** [review]
@gabriellpr I see you implemented the hover suggestion.
There's one caveat with the chosen approach (css hover etc): it only applies to desktop endpoints. The progress bar is still always visible on mobile endpoints.

My suggestion is to use the same hover/touch implementation as the other stuff (volume slider, full screen button, reload button) by moving the progress bar HTML block inside the `hover-toolbar-external-video` div. Further adjustments to CSS to the progressBar style will probably be necessary if that's done.

That being said it can be adjustment in a subsequent PR if need be, not a real blocker IMO.

**Comment by antobinary** [comment]
Let's do the iteration on 2.5+
